gio/ginetaddress.c

```diff
@@ -602,13 +628,19 @@ g_inet_address_new_any (GSocketFamily family)
   g_return_val_if_fail (G_INET_ADDRESS_FAMILY_IS_VALID (family), NULL);
 
   if (family == AF_INET)
-    {    
+    {
```

Comment:
Unnecessary change. (Even if technically an improvement, it makes the patch larger.)